src/yaml-languange-support/schema-utils.ts

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * An utility function for splitting text into lines.
+ */
+export function splitLines(text) {
+    return text.split(/\r\n?|\n/);
+} 
+
+/**
+ * A very easy utility function for decides whether the text is support to be a kubernetes yaml file, further
+ * implementations is needed to be more precise on the result rather than the regex test.
+ */
+export function isKubernetesYamlFile(text) {
+    return !!(/^kind\s*\:/g.exec(text) ||
+        /[^a-zA-Z]kind\s*\:/g.exec(text) ||
```

Comment:
What is the intent of the `[^a-zA-Z]` for here?  If the intention is to allow `  kind:` (indented) or `- kind:` or something like that then can we be more specific in the regex?

Reply:
I would like to match `kind:` and not `akind:`, I have updated the regex in newer PR.